api/src/test/java/org/openmrs/module/insuranceclaims/InsuranceClaimsServiceTest.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.insuranceclaims.api;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.openmrs.User;
+import org.openmrs.api.UserService;
+import org.openmrs.module.insuranceclaims.Item;
+import org.openmrs.module.insuranceclaims.api.dao.InsuranceClaimsDao;
+import org.openmrs.module.insuranceclaims.api.impl.InsuranceClaimsServiceImpl;
+
+import static org.hamcrest.Matchers.hasProperty;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+/**
+ * This is a unit test, which verifies logic in InsuranceClaimsService. It doesn't extend
+ * BaseModuleContextSensitiveTest, thus it is run without the in-memory DB and Spring context.
+ */
+public class InsuranceClaimsServiceTest {
+	
+	@InjectMocks
+	InsuranceClaimsServiceImpl basicModuleService;
```

Comment:
add private everywhere

Reply:
It is the test for generated service. I will change that in the separate PR where I will add the specific POJO classes. Thanks.